scene/gui/link_button.cpp

```diff
@@ -139,6 +150,13 @@ Size2 LinkButton::get_minimum_size() const {
 	return text_buf->get_size();
 }
 
+void LinkButton::launch_url() const {
```

Comment:
I think `open_url()` would be a better name as it'd be more consistent with `shell_open()`. Make sure to also rename the exposed method and update the class reference.

Also, is there much point in exposing `launch_url()`? If you want to trigger the button programmatically, it's better to assign a ShortCut resource to the button instead.